[2024-03-24T16:30:09.403+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tomtom.import_tomtom_data_to_csv scheduled__2024-03-19T00:00:00+00:00 [queued]>
[2024-03-24T16:30:09.467+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tomtom.import_tomtom_data_to_csv scheduled__2024-03-19T00:00:00+00:00 [queued]>
[2024-03-24T16:30:09.492+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-24T16:30:09.668+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): import_tomtom_data_to_csv> on 2024-03-19 00:00:00+00:00
[2024-03-24T16:30:09.734+0000] {standard_task_runner.py:60} INFO - Started process 366 to run task
[2024-03-24T16:30:09.805+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'tomtom', 'import_tomtom_data_to_csv', 'scheduled__2024-03-19T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/tomtom_dag.py', '--cfg-path', '/tmp/tmphts4ew5g']
[2024-03-24T16:30:09.819+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask import_tomtom_data_to_csv
[2024-03-24T16:30:09.900+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-24T16:30:10.133+0000] {task_command.py:423} INFO - Running <TaskInstance: tomtom.import_tomtom_data_to_csv scheduled__2024-03-19T00:00:00+00:00 [running]> on host cc0f04bb20f0
[2024-03-24T16:30:11.154+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tomtom' AIRFLOW_CTX_TASK_ID='import_tomtom_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-03-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-19T00:00:00+00:00'
[2024-03-24T16:30:11.157+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-24T16:30:11.159+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/tomtom_ingestion.py --date 2024-03-19']
[2024-03-24T16:30:11.212+0000] {subprocess.py:86} INFO - Output:
[2024-03-24T16:30:23.980+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-24T16:30:24.174+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=tomtom, task_id=import_tomtom_data_to_csv, execution_date=20240319T000000, start_date=20240324T163009, end_date=20240324T163024
[2024-03-24T16:30:24.254+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-24T16:30:24.370+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-24T16:50:24.086+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tomtom.import_tomtom_data_to_csv scheduled__2024-03-19T00:00:00+00:00 [queued]>
[2024-03-24T16:50:24.104+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tomtom.import_tomtom_data_to_csv scheduled__2024-03-19T00:00:00+00:00 [queued]>
[2024-03-24T16:50:24.106+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-24T16:50:24.207+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): import_tomtom_data_to_csv> on 2024-03-19 00:00:00+00:00
[2024-03-24T16:50:24.254+0000] {standard_task_runner.py:60} INFO - Started process 259 to run task
[2024-03-24T16:50:24.274+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'tomtom', 'import_tomtom_data_to_csv', 'scheduled__2024-03-19T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/tomtom_dag.py', '--cfg-path', '/tmp/tmptqqtw3y6']
[2024-03-24T16:50:24.291+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask import_tomtom_data_to_csv
[2024-03-24T16:50:24.468+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-24T16:50:25.062+0000] {task_command.py:423} INFO - Running <TaskInstance: tomtom.import_tomtom_data_to_csv scheduled__2024-03-19T00:00:00+00:00 [running]> on host 1ef4dfe662c6
[2024-03-24T16:50:26.192+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tomtom' AIRFLOW_CTX_TASK_ID='import_tomtom_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-03-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-19T00:00:00+00:00'
[2024-03-24T16:50:26.197+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-24T16:50:26.211+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/tomtom_ingestion.py --date 2024-03-19']
[2024-03-24T16:50:26.347+0000] {subprocess.py:86} INFO - Output:
[2024-03-24T16:50:39.061+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-24T16:50:39.245+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=tomtom, task_id=import_tomtom_data_to_csv, execution_date=20240319T000000, start_date=20240324T165024, end_date=20240324T165039
[2024-03-24T16:50:39.424+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-24T16:50:39.510+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-24T17:28:52.654+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tomtom.import_tomtom_data_to_csv scheduled__2024-03-19T00:00:00+00:00 [queued]>
[2024-03-24T17:28:52.729+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tomtom.import_tomtom_data_to_csv scheduled__2024-03-19T00:00:00+00:00 [queued]>
[2024-03-24T17:28:52.746+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-24T17:28:52.894+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): import_tomtom_data_to_csv> on 2024-03-19 00:00:00+00:00
[2024-03-24T17:28:52.927+0000] {standard_task_runner.py:60} INFO - Started process 292 to run task
[2024-03-24T17:28:52.959+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'tomtom', 'import_tomtom_data_to_csv', 'scheduled__2024-03-19T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/tomtom_dag.py', '--cfg-path', '/tmp/tmpt8m_omh6']
[2024-03-24T17:28:52.977+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask import_tomtom_data_to_csv
[2024-03-24T17:28:53.117+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-24T17:28:53.438+0000] {task_command.py:423} INFO - Running <TaskInstance: tomtom.import_tomtom_data_to_csv scheduled__2024-03-19T00:00:00+00:00 [running]> on host 77fbfb9df273
[2024-03-24T17:28:54.702+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tomtom' AIRFLOW_CTX_TASK_ID='import_tomtom_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-03-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-19T00:00:00+00:00'
[2024-03-24T17:28:54.711+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-24T17:28:54.713+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/tomtom_ingestion.py --date 2024-03-19']
[2024-03-24T17:28:54.766+0000] {subprocess.py:86} INFO - Output:
[2024-03-24T17:29:04.664+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-24T17:29:04.790+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=tomtom, task_id=import_tomtom_data_to_csv, execution_date=20240319T000000, start_date=20240324T172852, end_date=20240324T172904
[2024-03-24T17:29:05.005+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-24T17:29:05.076+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
